tldw_Server_API/tests/Flashcards/test_flashcards_endpoint_integration.py

```
@@ -2855,3 +2869,227 @@ def test_bulk_create_with_invalid_deck_returns_400(client_with_flashcards_db: Te
+    deckless_card = client_with_flashcards_db.post(
+        "/api/v1/flashcards",
+        json={"front": "Deckless card", "back": "Answer", "tags": ["deckless-tag"]},
+        headers=AUTH_HEADERS,
+    )
+    assert deckless_card.status_code == 200
+
+    keyword_table = flashcards_db._map_table_for_backend("keywords")
+    with flashcards_db.transaction() as conn:
+        conn.execute("UPDATE flashcards SET deleted = 1 WHERE uuid = ?", (deleted_card_uuid,))
+        conn.execute(
+            f"UPDATE {keyword_table} SET deleted = 1 WHERE keyword = ?",
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)